huaweicloud/services/modelarts/resource_huaweicloud_modelarts_service.go

```diff
@@ -0,0 +1,1007 @@
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `The description of the service.`,
+			},
+			"pool_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the new dedicated resource pool.`,
+			},
+			"vpc_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `The VPC ID to which a real-time service instance is deployed.`,
+			},
+			"subnet_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `The subnet ID.`,
+			},
+			"security_group_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `The security group ID.`,
+			},
+			"schedule": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Elem:     modelartsServiceScheduleSchema(),
+				Optional: true,
+				Computed: true,
+			},
+			"additional_properties": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Elem:     modelartsServiceAdditionalPropertySchema(),
+				Optional: true,
+				Computed: true,
+			},
+			"change_status_to": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Which status you want to change the service to, the valid value can be **running** or **stopped**.`,
+			},
+			"owner": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `User to which a service belongs`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Service status.`,
+			},
+			"access_address": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Access address of an inference request.`,
+			},
+			"bind_access_address": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Request address of a custom domain name.`,
+			},
+			"invocation_times": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Total number of service calls.`,
+			},
+			"failed_times": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Number of failed service calls.`,
+			},
+			"is_shared": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: `Whether a service is subscribed.`,
+			},
+			"shared_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Number of subscribed services.`,
+			},
+			"debug_url": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Online debugging address of a real-time service.`,
+			},
+			"is_free": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: `Whether a free-of-charge flavor is used.`,
+			},
+			"publish_at": {
```

Comment:
the following four arguments are hard-understanding. suggest to convert them to UTC format or delete them.

Reply:
deleted